05/app.js

```diff
@@ -1 +1,20 @@
-const numbers = [1, 2, 3, 4, 5, 6, 7];
\ No newline at end of file
+const numbers = [1, 2, 3, 4, 5, 6, 7];
+const evenTotal = sumEven(numbers);
+
+function sumEven(arr) {
+  return arr.filter(isEven).reduce(function(a, b) {return a + b})
+}
+
+function isEven(num) {
+  return num % 2 === 0
+}
+
+console.log(evenTotal)
+
+// Teoretycznie można powyższe skrócić jeszcze bardziej, tylko tracimy możliwość wykorzystania funkcji isEven gdzieś indziej (zmniejsza się też czytelność :P):
+
+function sumEven(arr) {
+  return arr.filter(function(num) {return num % 2 === 0}).reduce(function(a, b) {return a + b})
```

Comment:
Tutaj też można stawić isEven :)
Przy tak prostych obliczeniach wykorzystał bym to rozwiązanie.
Pamiętaj też żeby uważać na puste tablice `.reduce(function(a, b) {return a + b}, 0)` <- dodaj zero jako drugi parametr

Reply:
Jak wstawię isEven w drugim przykładzie, to stanie się identyczny jak pierwszy, a chodziło mi o przedstawienie dwóch wersji ;D